packages/react/src/xcss-prop/index.ts

```diff
@@ -57,8 +60,8 @@ type CompiledPropertyDeclarationReference = {
 export type CompiledStyles<TObject> = {
   [Q in keyof TObject]: TObject[Q] extends Record<string, unknown>
     ? CompiledStyles<TObject[Q]>
-    : CompiledPropertyDeclarationReference;
-};
+    : TObject[Q];
+} & CompiledPropertyDeclarationReference;
```

Comment:
I actually made this change in my PR when I messaged you last night but made it also a union of the object values so it could still be used in inline Xcss